provisioner/helper_kernel_nfs_server.go

```diff
@@ -204,13 +205,17 @@ func (p *Provisioner) createDeployment(nfsServerOpts *KernelNFSServerOptions) er
 				WithContainerBuildersNew(
 					container.NewBuilder().
 						WithName("nfs-server").
-						WithImage("itsthenetwork/nfs-server-alpine").
+						WithImage("openebs/nfs-server-alpine:latest").
```

Comment:
@watcher00090 - can this be changed to `openebs/nfs-server-alpine-amd64:ci` -- this is the only one that is available now under the docker hub. 

https://hub.docker.com/repository/docker/openebs/nfs-server-alpine-amd64/tags?page=1&ordering=last_updated

@mynktl - can probably work another follow-up PR to this one, that actually makes the image name configurable as mentioned in https://github.com/openebs/dynamic-nfs-provisioner/issues/18#issuecomment-830625044

Reply:
@kmova Yes. I just made the change.